lib/keycloak/credential_loader.sh

```diff
@@ -3,4 +3,8 @@
 # Load credentials from secrets
 export KEYCLOAK_ADMIN_PASSWORD=$(< /run/secrets/keycloak-admin-password)
 
-exec /opt/keycloak/bin/kc.sh "$@"
\ No newline at end of file
+if [ "$CANDIG_DEBUG_MODE" = 1 ]; then
```

Comment:
We should create a `CANDIG_PRODUCTION_MODE` entry in example.env and then use that here instead, since it's possible (maybe?) to want to debug a production server.